knotx-core/src/main/java/io/knotx/dataobjects/Fragment.java

```diff
@@ -79,10 +88,17 @@ public Fragment content(String content) {
     return this;
   }
 
+  /**
+   * @return a {@link JsonObject} containing context of processing this fragment, e.g. ActionKnot
```

Comment:
containing the data produced while Knots chain processing and used to exchange the context between them